import-scripts/clone_db_wrapper.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+
+# Script for running pre-import steps
+# Consists of the following:
+# - Determine which database is "production" vs "not production"
+# - Drop tables in the non-production database
+# - Clone the production database into the non-production database
+
+PORTAL_SCRIPTS_DIRECTORY=$1
+if [ -z $PORTAL_SCRIPTS_DIRECTORY ]; then
+    PORTAL_SCRIPTS_DIRECTORY="/data/portal-cron/scripts"
+fi
+if [ ! -f $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh ] ; then
+    echo "`date`: Unable to locate automation_env, exiting..."
+    exit 1
+fi
+source $PORTAL_SCRIPTS_DIRECTORY/automation-environment.sh
+
+# Create tmp directory for processing
+tmp=$PORTAL_HOME/tmp/import-cron-genie
+if ! [ -d "$tmp" ] ; then
+    if ! mkdir -p "$tmp" ; then
+        echo "Error : could not create tmp directory '$tmp'" >&2
+        exit 1
+    fi
+fi
+if [[ -d "$tmp" && "$tmp" != "/" ]]; then
+    rm -rf "$tmp"/*
+fi
+
+# This file will be used throughout entire import job to determine directionality
+SET_UPDATE_PROCESS_SCRIPT_FILEPATH="$PORTAL_SCRIPTS_DIRECTORY/set_update_process_state.sh"
+MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH="$PORTAL_SCRIPTS_DIRECTORY/airflowdb.properties"
+DROP_TABLES_FROM_MYSQL_DATABASE_SCRIPT_FILEPATH="$PORTAL_SCRIPTS_DIRECTORY/drop_tables_in_mysql_database.sh"
+CLONE_MYSQL_DATABASE_SCRIPT_FILEPATH="$PORTAL_SCRIPTS_DIRECTORY/clone_mysql_database.sh"
+
+# Update the process status database
+if ! $SET_UPDATE_PROCESS_SCRIPT_FILEPATH $MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH running ; then
+    echo "Error during execution of $SET_UPDATE_PROCESS_SCRIPT_FILEPATH : could not set running state" >&2
+    exit 1
+fi
+
+# Get the current production database color
+current_production_database_color=$(sh $PORTAL_SCRIPTS_DIRECTORY/get_database_currently_in_production.sh $MANAGE_DATABASE_TOOL_PROPERTIES_FILEPATH)
```

Comment:
To be consistent with usage above, the filepath to get_database_currently_in_production.sh should be stored in an environment variable rather than being constructed on the fly in this line.

Reply:
done!